interfaces/matlab_experimental/Reactor/ReactorNet.m

```diff
@@ -54,49 +54,56 @@
     methods
         %% ReactorNet Class Constructor
 
-        function r = ReactorNet(reactors)
+        function n = ReactorNet(reactors)
             % Create a :mat:class:`ReactorNet` object.
 
             ctIsLoaded;
 
-            if nargin ~= 1
-                error('Wrong number of arguments to ReactorNet constructor.');
+            if nargin == 0
+                reactors = {};
             end
 
             if isa(reactors, 'Reactor')
                 % Allow simpler syntax for creating a network with one
                 % reactor.
-                reactors = {reactor};
+                reactors = {reactors};
             end
 
-            r.id = ctFunc('reactornet_new');
+            n.id = ctFunc('reactornet_new');
 
             % add reactors
             nr = length(reactors);
-
             for i = 1:nr
-                ctFunc('reactornet_addreactor', r.id, reactors{i}.id);
+                ctFunc('reactornet_addreactor', n.id, reactors{i}.id);
             end
 
+            n.time = 0;
+
         end
 
         %% ReactorNet Class Destructor
 
-        function delete(r)
+        function delete(n)
             % Delete the :mat:class:`ReactorNet` object object.
 
-            if isempty(r.id)
+            if isempty(n.id)
                 return
             end
-            ctFunc('reactornet_del', r.id);
+            ctFunc('reactornet_del', n.id);
         end
 
         %% ReactorNet Utility Methods
 
-        function advance(r, tout)
+        function addReactor(n, r)
```

Comment:
This won't be supported by the new CLib API. Instead, all reactors have to be passed to the constructor.